test/run_test.py

```diff
@@ -64,7 +64,7 @@
     'test_type_promotion',
     'test_jit_disabled',
     'test_function_schema',
-    'test_overrides',
```

Comment:
nit: can we revert this line as well ;) ?

Reply:
Shall I keep the trailling comma?